tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Do we need `es5` here? We can focus on Node.js 10 for testing

Reply:
This is the default for TS. I think you can safely use esnext.

Reply:
Let's remove it from the test

Reply:
If the TS takes root, the config can be moved to [`sharec-config`](https://github.com/logux/sharec-config).